python/LoadExternalSTAC/LoadLandsatSTAC.ipynb

```diff
@@ -0,0 +1,890 @@
e/GwcPHkShUMBdd92VVDcSR9rjIsLxxx+P5557LlYZWSDtsZk6dSoKhQKz0CxduhQA8NJLL8VrfIrIcs7ceeedOP/883075o2KtMfmjjvuQHd3Nz7zmc9g9erVOPXUU/Htb38bDzzwAP7whz8k1Y3Ekfa4WJaFT3/60zh8+DBefPFF7NixwyVZmTdvXiJ9SAtJjM2f//xnnHnmmXjPe96Dm2++mTk2ZcoUH0vhzp070dXVhdbW1mQ7kyDSHpexjKzG5rvf/S6uvvpqfP/73/e5ezYishqXuXPnYsWKFbjmmmvw13/91/jEJz6RdFcSR5pj09/fj0ceeQTXX3+9KwP//d//PTZs2IBCoYAHH3ww1b75UJfIqiMI1WrVvu666+xp06bZmzdv9h0ngXH/+Z//6f729NNPRwr0f/LJJ93/P/WpT9mdnZ32k08+ae/bty/RPiWBrMZFhKuvvtpevXp15LanjazG5uc//7kNwH7uuefc3wgRwjPPPJNMZxJE1nNm69attmVZ9v/8z/8k0v40kdXY3HjjjfZxxx3H/LZt2zYbgP3b3/42fkcSRj3Xmcsuu8xet25d5LanjaTGZuPGjfakSZPsD33oQ8J6PvzhD9tHHXUU89sll1zSsOQQWY0LjbFCDpHl2Nxzzz12S0uL/aMf/SjZTqSAeswZgltvvdWePXt2rPaniSzGplKpMPLvk08+aV977bX24sWL7SeffNI+fPhwOp2TwChOMXHttdfa3d3d9sMPP2xv377d/X9wcNA9533ve589a9Ys+8EHH7QfeeQRe926db4X7rPPPms/9thj9nvf+1570aJF9mOPPWY/9thj9sjIiLDeRmfVy2pc7r77bvuee+6xN23aZG/atMn+h3/4BzuXy9l33XVXpv0Ng6zGplKp2GvWrLFPPfVUu6+vz37kkUfs448/3j777LMz7a8usn6Wbr75ZnvatGl2uVzOpH9xkNXYPPDAA7ZlWfatt95qb9682X700Uftc8891549ezZTV6Mgq3HZvXu3/a//+q/2pk2b7Mcee8z+q7/6K7ulpcX+wx/+kGl/wyCJsXnyySftiRMn2u9617uYMnbt2uWes3XrVrutrc3+0Ic+ZG/atMm+44477Hw+b993332Z9lcXWY2LbdvuPFq7dq196aWX2o899pj91FNPZdbXsMhqbL7zne/YhULBvuOOO5hzDhw4kGl/dZHVuHz5y1+2f/zjH9ubN2+2N2/ebN955512Z2en/bGPfSzT/oZBls8TjXqy6hnFKSYACP//+te/7p4zNDRkv//977d7e3vttrY2+61vfau9fft2ppzTTjtNWM7zzz8vrLfRFaesxuXuu++2ly5dare1tdldXV32cccdx9BeNiKynDOvvvqq/ba3vc3u6OiwJ0+ebF955ZX23r17M+ppOGQ5LpVKxZ4xY4b90Y9+NKPexUOWY/Mf//Ef9urVq+329nZ74sSJ9pvf/GZ706ZNGfU0HLIal927d9snnHCC3d7ebre1tdlnnnmm/fvf/z7DnoZHEmNzyy23CMvgd8Afeughe9WqVXZTU5M9b948po5GQ5bjonNOIyGrsZE9b1dccUV2nQ2BrMblS1/6kr18+XJXnlm9erX9la98hUkP0WjI8nmiUU/FybJt24aBgYGBgYGBgYGBgYGBFIYcwsDAwMDAwMDAwMDAIABGcTIwMDAwMDAwMDAwMAiAUZwMDAwMDAwMDAwMDAwCYBQnAwMDAwMDAwMDAwODABjFycDAwMDAwMDAwMDAIABGcTIwMDAwMDAwMDAwMAiAUZwMDAwMDAwMDAwMDAwCYBQnAwMDAwMDAwMDAwODABjFycDAwMDgiMWVV16JCy64oN7NMDAwMDA4AlCodwMMDAwMDAyiwLIs5fFbbrkFX/ziF2HbdkYtMjAwMDA4kmEUJwMDAwODMYnt27e7n7/3ve/h4x//OJ555hn3t46ODnR0dNSjaQYGBgYGRyCMq56BgYGBwZjElClT3P+7u7thWRbzW0dHh89V73Wvex0+8IEP4IYbbkBvby8mT56Mr33taxgYGMBVV12Fzs5OLFiwAD/72c+YujZu3Ig3vOEN6OjowOTJk3HZZZdhz549GffYwMDAwKCeMIqTgYGBgcFrCt/4xjcwYcIE/PGPf8QHPvABXHvttbjoootw4oknoq+vD+eccw4uu+wyDA4OAgAOHDiAM844A6tXr8YjjzyC++67Dzt37sQ73vGOOvfEwMDAwCBLGMXJwMDAwOA1hZUrV+Lmm2/GwoULcdNNN6GlpQUTJkzANddcg4ULF+LjH/849u7diyeeeAIA8OUvfxmrV6/GP/7jP2LJkiVYvXo17rrrLjz00EPYvHlznXtjYGBgYJAVTIyTgYGBgcFrCkcffbT7OZ/PY/z48VixYoX72+TJkwEAu3btAgBs2LABDz30kDBeasuWLVi0aFHKLTYwMDAwaAQYxcnAwMDA4DWFYrHIfLcsi/mNsPVVq1UAwOHDh/GmN70Jn/70p31lTZ06NcWWGhgYGBg0EoziZGBgYGBgoMCaNWvwgx/8AHPmzEGhYF6bBgYGBq9VmBgnAwMDAwMDBa677jrs27cPl1xyCf70pz9hy5Yt+PnPf46rrroKlUql3s0zMDAwMMgIRnEyMDAwMDBQYNq0afjtb3+LSqWCc845BytWrMANN9yAnp4e5HLmNWpgYGDwWoFlm5TqBgYGBgYGBgYGBgYGSpitMgMDAwMDAwMDAwMDgwAYxcnAwMDAwMDAwMDAwCAARnEyMDAwMDAwMDAwMDAIgFGcDAwMDAwMDAwMDAwMAmAUJwMDAwMDAwMDAwMDgwAYxcnAwMDAwMDAwMDAwCAARnEyMDAwMDAwMDAwMDAIgFGcDAwMDAwMDAwMDAwMAmAUJwMDAwMDAwMDAwMDgwAYxcnAwMDAwMDAwMDAwCAARnEyMDAwMDAwMDAwMDAIwP8PQBGb7XyvbzMAAAAASUVORK5CYII=",
```

Comment:
very good, maybe good to point out that in this case we did not harmonize the sentinel2 and landsat collection; explaining the difference in amplitude. 

There is however also a harmonized dataset on MPC